test/components/StatusAlert.spec.js

```diff
@@ -0,0 +1,69 @@
+import { createLocalVue, mount } from '@vue/test-utils'
+import { BootstrapVue } from 'bootstrap-vue'
+import StatusAlert from '@/components/StatusAlert'
+import statuses from '~/modules/statuses'
+
+const localVue = createLocalVue()
+localVue.use(BootstrapVue)
+
+describe('StatusAlert.vue', () => {
+  let wrapper
+
+  it('has an alertMessage', () => {
+    wrapper = mount(StatusAlert, {
+      localVue,
+      data() {
+        return { alertMessage: 'Alert Message' }
+      },
+    })
+
+    expect(wrapper.vm.alertMessage).toBe('Alert Message')
+  })
```

Comment:
Maybe an expect saying default is Status.Idle?

Reply:
Yes good idea.  I've added a test for the default state of all the data elements now.